yacron/config.py

```diff
@@ -241,6 +244,18 @@ class ConfigError(Exception):
         Opt("jobs"): Seq(Map(_job_schema_dict)),
         Opt("web"): Map({"listen": Seq(Str())}),
         Opt("include"): Seq(Str()),
+        Opt("logging"): Map(
```

Comment:
In favor of future upgrades, could there by just `YamlAny`?

Reply:
Hah, yes, and no.  I started doing just that, before discovering a "unfun" fact about scrityaml.  When you use the "any" validator, all yaml values become strings.  In this particular example, the `version` field was a string, and python's logging didn't like that, it has to be an int.  Took me a while to understand that nuance.  Declaring the field as `Int()` fixes that.

The nested objects are declared YamlAny, which _seems_ to work so far, but I fear the string-ification issue might rear its ugly head again in the future :confused: